ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/425__model_serving_ovms_on_kserve.robot

```diff
@@ -0,0 +1,148 @@
+*** Settings ***
+Library           OperatingSystem
+Library           ../../../../libs/Helpers.py
+Resource          ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource          ../../../Resources/OCP.resource
+Resource          ../../../Resources/CLI/ModelServing/modelmesh.resource
+Suite Setup       Model Serving Suite Setup
+Suite Teardown    Model Serving Suite Teardown
+Test Tags         ModelMesh
+
+
+*** Variables ***
+${INFERENCE_INPUT}=    @ods_ci/tests/Resources/Files/modelmesh-mnist-input.json
+${INFERENCE_INPUT_OPENVINO}=    @ods_ci/tests/Resources/Files/openvino-example-input.json
+${PRJ_TITLE}=    model-serving-project
+${PRJ_DESCRIPTION}=    project used for model serving tests
+${MODEL_CREATED}=    ${FALSE}
+${MODEL_NAME}=    test-model
+${RUNTIME_NAME}=    Model Serving Test
+${SECOND_PROJECT}=    sec-model-serving-project
+${SECURED_MODEL}=    test-model-secured
+${SECURED_RUNTIME}=    Model Serving With Authentication
+${EXPECTED_INFERENCE_SECURED_OUTPUT}=    {"model_name":"${SECURED_MODEL}__isvc-83d6fab7bd","model_version":"1","outputs":[{"name":"Plus214_Output_0","datatype":"FP32","shape":[1,10],"data":[-8.233053,-7.7497034,-3.4236815,12.3630295,-12.079103,17.266596,-10.570976,0.7130762,3.321715,1.3621228]}]}
+${EXPECTED_INFERENCE_OUTPUT_OPENVINO}=    {"model_name":"${MODEL_NAME}__isvc-8655dc7979","model_version":"1","outputs":[{"name":"Func/StatefulPartitionedCall/output/_13:0","datatype":"FP32","shape":[1,1],"data":[0.99999994]}]}
+
+
+*** Test Cases ***
+Verify Openvino_IR Model Via UI (Kserve)
+    [Documentation]    Test the deployment of an openvino_ir model
+    [Tags]    Smoke
+    ...    ODS-2054    ODS-2053
+    Create Openvino Models For Kserve    server_name=${RUNTIME_NAME}    model_name=${MODEL_NAME}
+    ...    project_name=${PRJ_TITLE}   num_projects=1
+    Run Keyword And Continue On Failure    Verify Model Inference    ${MODEL_NAME}    ${INFERENCE_INPUT_OPENVINO}
+    ...    ${EXPECTED_INFERENCE_OUTPUT_OPENVINO}    token_auth=${FALSE}
+    Clean All Models Of Current User
```

Comment:
better to run this in the teardown because if the test fails in the middle of first keyword execution we may not clean up

Reply:
Done